### DISCUSSION TITLE
How can I update my Microsoft learn student ambassador profile with my github account

### DISCUSSION THREAD

**Discussion by Rnch268** [microsoft/SATechnicalOnboarding#94] [Questions]
when I already create my githup account what to do next how can i update my profile with my githup account.

**Comment by AnthonyByansi** [comment]
Hi @Rnch268 
Login to the website and click get started---> Profile-->Edit--->Social Media-->add your github handle.
![profile](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/f474a403-c4bb-488f-9168-e2f421e0bbbb)
![profile2](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/6bc814eb-480c-4d3d-9cef-4870b26d1c2c)

**Comment by PranjaliBhardwaj** [comment]
Hello @Rnch268 Login to your student ambassador account and then go to profile , then go to edit profile, and add account, add github section in social Media, paste your Github URL and then save it.
Hope it helps!
